addons/hr_recruitment/views/hr_applicant_views.xml

```diff
@@ -5,11 +5,12 @@
         <field name="name">Applicants</field>
         <field name="model">hr.applicant</field>
         <field name="arch" type="xml">
-            <tree string="Applicants" multi_edit="1" sample="1">
+            <tree string="Applicants" multi_edit="1" sample="1" decoration-danger="application_status == 'refused'">
```

Comment:
Given the specs, shouldn't it be `decoration-muted` instead?
```suggestion
            <tree string="Applicants" multi_edit="1" sample="1" decoration-muted="application_status == 'refused'">
```

Reply:
went over it with Geoffrey, text-muted does not stand out as it's the standard text in this view since milk, so we use danger.

Reply:
Noted! I adapted the specs of the task for future documentation :ok_hand: